### PR TITLE
Metalink converter

### PR DESCRIPTION
**Pull request by huard**
## Overview

Changes:

* Added metalink 3 converter

Requires `pymetalink` package.

### CODE REVIEW

**Comment by tomLandry** [comment]
Talk about relevant work. Cool stuff David, thanks. We'll see how/when to weave that back into the mix.

**Comment by huard** [comment]
It still needs to be integrated in pywps. There is a branch but it still needs some love. Good news is, it looks overall like a viable solution.

**Comment by cehbrecht** [comment]
@huard probably need to merge master into this PR (merged PR #111 before).

**Comment by huard** [comment]
@cehbrecht Done

File: birdy/client/converters.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 class BaseConverter(object):
     mimetype = None
+    extensions = []
```
Comment (cehbrecht) [inline birdy/client/converters.py]: is extensions used?

File: requirements_extra.txt [hunk 2]
```diff
@@ -2,4 +2,5 @@ netCDF4
 ipywidgets
 ipython
 geojson
+pymetalink
```
Comment (cehbrecht) [inline requirements_extra.txt]: pymetalink only available on GitHub? I haven't seen a package on pypi or conda.

File: birdy/client/converters.py (hunk 1 above)
Comment (huard) [reply birdy/client/converters.py]: It's used in PR #114.

File: requirements_extra.txt (hunk 2 above)
Comment (huard) [reply requirements_extra.txt]: That's my understanding. It does not seem to be very active, but it works.

Comment (cehbrecht) [reply requirements_extra.txt]: You could add the pip+github install in the conda `environment.yml`.

**Comment by cehbrecht** [comment]
@huard feel free to merge.

File: requirements_extra.txt (hunk 2 above)
Comment (huard) [reply requirements_extra.txt]: Done.

**Pull request merged**